README.md

```diff
@@ -114,7 +114,48 @@ iterate over the variables and their values using the cartesian product
 results in a format that is friendly with plotting tools.
 
 
-## Getting Started
+## Quick start with sugars
+
+If you want to rapidly evaluate command-line programs without writing full
+benchmark definitions, `benchkit` offers a minimal interface via the
+`benchkit.quick` module.
+
+For example, you can benchmark `dd` with 2 parameters:
```

Comment:
I would suggest giving another name. `dd` reminds me too much of the `dd` command in Linux.